ui/partials/perf/comparechooserctrl.html

```diff
@@ -12,7 +12,7 @@
           <label for="original-revision-input">Revision</label>
           <div class="form-group has-error">
             <input id="original-revision-input" maxlength="12" class="form-control" ng-model="originalRevision">
-            <br/>
+            <br>
```

Comment:
<br/> indicates a singular element (vs. <br> which implies there is a closing </br> somewhere, which there isn't), we want to keep that syntax. Please take out this change -- it should actually be the input element which has a closing "/" on it, in fact. i.e.:

    <input id="original-revision-input" maxlength="12" class="form-control" ng-model="originalRevision"/>